server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/qualityprofile/ActiveRulesHolderImplTest.java

```diff
@@ -48,7 +48,7 @@ public void get_inactive_rule() throws Exception {
 
   @Test
   public void get_active_rule() throws Exception {
-    underTest.set(asList(new ActiveRule(RULE_KEY, Severity.BLOCKER, Collections.<String, String>emptyMap())));
+    underTest.set(asList(new ActiveRule(RULE_KEY, Severity.BLOCKER, Collections.<String, String>emptyMap(), 1_000L)));
```

Comment:
we've come to use constants to avoid non descriptive magic numbers (and duplication, but that's not the main point).
Here, you could name it `SOME_DATE`